src/commands/kv/namespace/site.rs

```diff
@@ -0,0 +1,60 @@
+use cloudflare::endpoints::workerskv::create_namespace::CreateNamespace;
+use cloudflare::endpoints::workerskv::create_namespace::CreateNamespaceParams;
+use cloudflare::endpoints::workerskv::list_namespaces::ListNamespaces;
+use cloudflare::endpoints::workerskv::WorkersKvNamespace;
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::response::ApiFailure;
+
+use crate::commands::kv;
+use crate::settings::global_user::GlobalUser;
+use crate::settings::target::Target;
+use crate::terminal::message;
+
+pub fn site(target: &Target, user: &GlobalUser) -> Result<WorkersKvNamespace, failure::Error> {
```

Comment:
the purpose of this function is to create a specially-named namespace for storing the static assets for a particular worker. tl;dr - try to create a namespace with a title specified by convention. if it already exists, get its id by listing namespaces and finding the one with the matching title. return the namespace as a result.

Reply:
this would be an awesome comment to put in code in the future!

Reply:
putting it there now